Dockerfile.ly2video

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y \
+    timidity \
+    python-pip \
+    python-imaging swig \
+    libasound-dev \
+    git \
+    curl \
+    ffmpeg \
+    vim
+
+ARG BASE_PATH
+ARG SCORE_PATH
+
+# Check for mandatory build arguments
+RUN \
+    : "${BASE_PATH:?mandatory build argument is missing}"
+RUN \
+    : "${SCORE_PATH:?mandatory build argument is missing}"
+
+RUN mkdir -p ${BASE_PATH}/ly2video
+
+WORKDIR ${BASE_PATH}
+
+RUN curl -o lilypond.sh https://lilypond.org/download/binaries/linux-64/lilypond-2.20.0-1.linux-64.sh
+
+RUN ls -l
+
+RUN chmod +x lilypond.sh
+
+RUN sh lilypond.sh --batch
+
+WORKDIR ${BASE_PATH}/ly2video
+
+RUN git clone https://github.com/aspiers/ly2video.git .
```

Comment:
Please could you change it so that it uses `COPY` to copy the source from the current repo into the container, rather than doing a git clone?  This will allow building containers with local changes, or from pull request branches etc.

Reply:
Done!